docs/data/location-ids.md

```diff
@@ -16,22 +16,46 @@ When you [report an Encounter](report-encounter.md), you can set its **location
 
 ![match Criteria](../assets/images/wb-encounter-customalg.png)
 
-## Adding a New Study Site
+## Adding a Location ID
 
-To add a new study site, you'll need to contact your Wildbook's administrator or make the request at the [Wildbook Community](https://community.wildme.org).
+To add a new Location ID, you'll need to contact your Wildbook's administrator or make the request at the [Wildbook Community](https://community.wildme.org).
```

Comment:
should we add that they can make a PR against their platform branch? or handle that in a different page?

Reply:
Good call! I've added a link to our PR page as a third option.